<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSettings</class>
 <widget class="QDialog" name="DialogSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Camera</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboCamera"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Measure device port</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboSerialPort"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tolerances Standard</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="0">
        <widget class="FormRange" name="widgetRangeStandardP6_6" native="true"/>
       </item>
       <item row="1" column="1">
        <widget class="FormRange" name="widgetRangeStandardP7_2" native="true"/>
       </item>
       <item row="2" column="1">
        <widget class="FormRange" name="widgetRangeStandardP7_3" native="true"/>
       </item>
       <item row="3" column="0">
        <widget class="FormRange" name="widgetRangeStandardP6_4" native="true"/>
       </item>
       <item row="1" column="0">
        <widget class="FormRange" name="widgetRangeStandardP6_2" native="true"/>
       </item>
       <item row="7" column="0">
        <widget class="FormRange" name="widgetRangeStandardP6_8" native="true"/>
       </item>
       <item row="4" column="0">
        <widget class="FormRange" name="widgetRangeStandardP6_5" native="true"/>
       </item>
       <item row="0" column="0">
        <widget class="FormRange" name="widgetRangeStandardP6_1" native="true"/>
       </item>
       <item row="6" column="0">
        <widget class="FormRange" name="widgetRangeStandardP6_7" native="true"/>
       </item>
       <item row="0" column="1">
        <widget class="FormRange" name="widgetRangeStandardP7_1" native="true"/>
       </item>
       <item row="2" column="0">
        <widget class="FormRange" name="widgetRangeStandardP6_3" native="true"/>
       </item>
       <item row="8" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tolerances Maxi</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="7" column="0">
        <widget class="FormRange" name="widgetRangeMaxiP6_8" native="true"/>
       </item>
       <item row="8" column="0">
        <widget class="FormRange" name="widgetRangeMaxiP6_9" native="true"/>
       </item>
       <item row="9" column="0">
        <widget class="FormRange" name="widgetRangeMaxiP6_10" native="true"/>
       </item>
       <item row="0" column="0">
        <widget class="FormRange" name="widgetRangeMaxiP6_1" native="true"/>
       </item>
       <item row="6" column="0">
        <widget class="FormRange" name="widgetRangeMaxiP6_7" native="true"/>
       </item>
       <item row="1" column="0">
        <widget class="FormRange" name="widgetRangeMaxiP6_2" native="true"/>
       </item>
       <item row="2" column="0">
        <widget class="FormRange" name="widgetRangeMaxiP6_3" native="true"/>
       </item>
       <item row="5" column="0">
        <widget class="FormRange" name="widgetRangeMaxiP6_6" native="true"/>
       </item>
       <item row="4" column="0">
        <widget class="FormRange" name="widgetRangeMaxiP6_5" native="true"/>
       </item>
       <item row="0" column="1">
        <widget class="FormRange" name="widgetRangeMaxiP7_1" native="true"/>
       </item>
       <item row="3" column="0">
        <widget class="FormRange" name="widgetRangeMaxiP6_4" native="true"/>
       </item>
       <item row="1" column="1">
        <widget class="FormRange" name="widgetRangeMaxiP7_2" native="true"/>
       </item>
       <item row="2" column="1">
        <widget class="FormRange" name="widgetRangeMaxiP7_3" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FormRange</class>
   <extends>QWidget</extends>
   <header>formrange.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
